src/main/java/org/reactome/qa/doisuggester/Suggester.java

```diff
@@ -0,0 +1,142 @@
+package org.reactome.qa.doisuggester;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.gk.model.GKInstance;
+
+/**
+ * An object that makes a DOI suggestion about an RLE
+ * @author sshorser
+ *
+ */
+public class Suggester
+{
+	/**
+	 * Represents a suggestion about a single pathway, and the RLEs and InstanceEdits that are related to the suggestion.
+	 * @author sshorser
+	 *
+	 */
+	class Suggestion
+	{
+		private String pathway;
+		private Map<String, Set<String>> rlesToInstanceEdits;
+
+		/**
+		 * Gets the pathway.
+		 * @return
+		 */
+		public String getPathway()
+		{
+			return pathway;
+		}
+
+		/**
+		 * Sets the pathway.
+		 * @param pathway
+		 */
+		public void setPathway(String pathway)
+		{
+			this.pathway = pathway;
+		}
+
+		/**
+		 * Gets a map that is a mapping of RLEs to a set of InstanceEdits.
+		 * @return
+		 */
+		public Map<String, Set<String>> getRlesToInstanceEdits()
+		{
+			return rlesToInstanceEdits;
+		}
+
+		/**
+		 * Sets a map that is a mapping of RLEs to a set of InstanceEdits.
+		 * @param rlesToInstanceEdits
+		 */
+		public void setRlesToInstanceEdits(Map<String, Set<String>> rlesToInstanceEdits)
+		{
+			this.rlesToInstanceEdits = rlesToInstanceEdits;
+		}
+	}
+
+	private ReactionlikeEvent rle;
+
+	/**
+	 * Create a new suggester.
+	 * @param reactionlikeEvent The RLE to make a suggestion about.
+	 */
+	public Suggester(ReactionlikeEvent reactionlikeEvent)
+	{
+		this.rle = reactionlikeEvent;
+	}
+
+	/**
+	 * Gets a suggestion. The suggestion is actually returned as a set. In the case where
+	 * an RLE has multiple parents and both may require a new DOI, then > 1 Suggestion object will be returned.
+	 * @return
+	 */
+	public Set<Suggestion> getSuggestion()
+	{
+		Set<Suggestion> suggestions = new HashSet<>();
+		Set<GKInstance> pathways = new HashSet<>();
+		Set<String> suggestedInstanceEdits = new HashSet<>();
+		// Handle RLEs that exist in previous versions of Reactome.
+		if (!this.rle.isNewRLE())
+		{
+			List<GKInstance> newInstanceEdits = ReactionlikeEvent.filterForNonReactomeInstanceEdits(this.rle.getNewInstanceEdits());
```

Comment:
This method is only used in this class and since it is static and doesn't refer to a `ReactionlikeEvent` object, maybe it makes more sense to belong to the Suggester class?